uint16 C_FULLSTOP=0
uint16 C_STOP=1
uint16 C_RUN=2

uint16 J_BASE=0
uint16 J_SHOULD=1
uint16 J_ELBOW=2
uint16 J_WRIST=3
uint16 J_ROTATE=4
uint16 J_GRAB=5
uint16 J_SCREW=6
uint16 J_LSRGRAB=7
uint16 J_LSRPHIL=8
uint16 J_LSRHEX=9

uint16 cmd
uint16 joint
int16 value
